### DISCUSSION TITLE
Global header parameter

### DISCUSSION THREAD

**Discussion by SzaboGabesz** [hussein4alaa/laravel-g4t-swagger-auto-generate#69] [Q&A] [answered]
Hi Hussein,

I would like to add a custom header parameter to each end-point, but I don't know how it's possible if possible at all. In my case this is the X-Tenant header to identify the tenant affected by the query. This header must be specified for every request. I investigated the code and as far as I see there is only one place where I can define parameter with header type and this is the security_schemes config, so I guess this is not possible at the moment. Or is it?

**Comment by hussein4alaa** [comment, marked as answer]
Hi @SzaboGabesz,

Thank you for your inquiry. 
Currently, the feature you are requesting is not available in the package.
I will work on adding this feature as soon as possible to meet your needs.

Thank you for your suggestion and patience

**Reply by SzaboGabesz** [reply]
Wow, this was fast :) Thank you very much for your work. Could you give me a rough estimation how much time this takes for you to finish?
